pkg/virt-handler/vm.go

```diff
@@ -1904,15 +1904,26 @@ func (d *VirtualMachineController) isPreMigrationTarget(vmi *v1.VirtualMachineIn
 }
 
 func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
+	err := d.checkMasquradeInterfacesForMigration(vmi)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (d *VirtualMachineController) checkMasquradeInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
```

Comment:
I can ignore it, but this looks really ugly and messy function.

Not sure why the original and this new one is a method and not a regular function, none are using the receiver.
The name is also odd, it looks to me live a "validate" or "assert", the current name of "check" has little meaning to me.

I would just refactor this to look like this:
```
podnet := podNetwork(vmi)
if podnet := podNetwork(vmi); podnet != nil && lookupIface(podnet.Name).Masquerade == nil {
    return fmt.Errorf(...)
}
```

But if you want to leave it like this, just change the name to something I can understand from reading it. Like: `validatePodNetworkInterfaceUsesMasqueradeBinding`

Reply:
I changed the function name and the code to what you suggest it does improve readability